community/bolt/src/main/java/org/neo4j/bolt/BoltKernelExtension.java

```diff
@@ -181,6 +181,10 @@ public Lifecycle newInstance( KernelContext context, Dependencies dependencies )
         {
             life.add( new NettyServer( scheduler.threadFactory( boltNetworkIO ), connectors ) );
             log.info( "Bolt Server extension loaded." );
+            for ( ProtocolInitializer connector : connectors )
+            {
+                logging.getUserLog( Sessions.class ).info( "Bolt enabled on %s.", connector.address() );
+            }
```

Comment:
This adds a log line for each (bolt) connector to the user log. We use (the absence) this in our test to assert that no connectors are being started.